rootfs-builder/centos/Dockerfile.am

```diff
@@ -0,0 +1,4 @@
+From centos:@OS_VERSION@
```

Comment:
Nice. Can you rename these files `Dockerfile.in` though (as the `.am` extension implies they are in automake format).